lib/model/QubitInformationObject.php

```diff
@@ -1168,6 +1168,21 @@ public function getDigitalObject()
   }
 
   /**
+   * Get the digital object's checksum value
+   *
+   * @return string  digital object checksum or null
+   */
+  public function getDigitalObjectChecksum()
+  {
+    if (null !== $do = $this->getDigitalObject())
+    {
+      return $do->getChecksum();
+    }
+
+    return null;
```

Comment:
Well, we didn't in my days as a developer.  I don't know what you crazy kids get up to these days. :stuck_out_tongue_winking_eye: